src/main/scala/io/iohk/ethereum/consensus/validators/std/StdOmmersValidator.scala

```diff
@@ -85,26 +91,28 @@ class StdOmmersValidator(blockchainConfig: BlockchainConfig, blockHeaderValidato
     * @param blockNumber    the number of the block to which the ommers belong
     * @param ommers         the list of ommers to validate
     * @param getNBlocksBack from where the ommers' parents will be obtained
-    *
-    * @return [[OmmersValidator.OmmersValid]] if valid, an [[OmmersValidator.OmmersError.OmmersUsedBeforeError]] otherwise
+    * @return [[OmmersValidator.OmmersValid]] if valid, an [[OmmersValidator.OmmersError.OmmerIsAncestorError]] otherwise
     */
-  private def validateOmmersAncestors(
+  private[std] def validateOmmersAncestors(
       parentHash: ByteString,
       blockNumber: BigInt,
       ommers: Seq[BlockHeader],
       getNBlocksBack: GetNBlocksBack
   ): Either[OmmersError, OmmersValid] = {
 
-    val ancestorsOpt = collectAncestors(parentHash, blockNumber, getNBlocksBack)
+    val ancestors = collectAncestors(parentHash, blockNumber, getNBlocksBack)
+    def ommersHashes: Seq[ByteString] = ommers.map(_.hash)
```

Comment:
Would it make sense to use `val` instead of `def`, and `lazy val` for some bindings that are not always used?

Reply:
`lazy val` can also be. Those pieces of code only run 0 or 1 time, def is just a more compact notation :)

Reply:
A future refactoring could inadvertently cause some of this to run 1+ times, though.

Reply:
I have updated it to `lazy val`. Future proof :)